0003-Disable-statx-for-all-builds.-JB-50106.patch

```diff
@@ -32,3 +45,6 @@ that gracefully without issues thanks to rust's type system.
  }
  
  cfg_has_statx! {{
+-- 
```

Comment:
This doesn't seem to be necessary

Reply:
well git format patch does it, so I would keep it

Reply:
up to you!